2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,8 +19,12 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+  let object = Object.fromEntries(countryCurrencyCodes);
+
+  return object;
 }
 
+// Note The Object.fromEntries() method transforms a list of key-value pairs into an object.
```

Comment:
nice exploration of an existing method to accomplish this, well done! have a think of how you would approach this if you had to do it manually